app/code/Bulbulatory/Recommendations/Setup/Patch/Data/InsertRecommendations.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Bulbulatory\Recommendations\Setup\Patch\Data;
+
+use Bulbulatory\Recommendations\Setup\Patch\Schema\UpgradeHashRecommendations;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+class InsertRecommendations implements DataPatchInterface
+{
+    /**
+     * @var ModuleDataSetupInterface
+     */
+    private $moduleDataSetup;
+
+    public function __construct(
+        ModuleDataSetupInterface $moduleDataSetup
+    ) {
+
+        $this->moduleDataSetup = $moduleDataSetup;
+    }
+
+    public function apply()
+    {
+        $this->moduleDataSetup->startSetup();
+        $setup = $this->moduleDataSetup;
+
+        $data = [
+            ['reference' => 'Very good site', 'email' => 'test_one@test_one.com'],
+            ['reference' => 'Very very good site!!!', 'email' => 'test_two@test_two.com']
+        ];
+        foreach ($data as $bind) {
+            $setup->getConnection()
+                ->insertForce($setup->getTable('bulbulatory_recommendations'), $bind);

```

Comment:
Here inserting data is ok, as it is a fake project for learning purposes but in real life you don't want to insert random data to the database - the tables should be filled only when a proper action is taken in the app.

Reply:
@TM18

> Here inserting data is ok, as it is a fake project for learning purposes but in real life you don't want to insert random data to the database - the tables should be filled only when a proper action is taken in the app.

Of course! I created Bulbulatory/Recommendations/Setup/Patch/Data/InsertRecommendations class for:

1. Check _bulbulatory_recommendations_ table
2. Gain experience with Data Patch